pages/api/recipes/[id].js

```diff
@@ -0,0 +1,32 @@
+import { data } from "@/lib/data";
+const APP_ID = process.env.APP_ID;
+const API_KEY = process.env.API_KEY;
+
+export default async function handler(request, response) {
+  console.log("Getting all recipes");
+  const { query } = request;
+  const { id } = query;
```

Comment:
Would be this possibile?
```suggestion
  const { id } = request.query;
```

Save one line ;-)